install_config/aggregate_logging.adoc

```diff
@@ -230,40 +240,28 @@ When setting the `*ENABLE_OPS_CLUSTER*` parameter to `*true*`, Fluentd splits
 logs between the Elasticsearch cluster and a cluster reserved for operations
 logs. This means a second Elasticsearch and Kibana are deployed. The deployments
 are distinguishable by the *-ops* included in their names.
-
-. Next, as a cluster administrator, deploy the template that is created by
-the deployer:
 +
-====
-----
-$ oc process logging-support-template | oc create -f -
-----
-====
-
-. Enable the Fluentd service account by editing the security context:
+This creates a deployer pod and prints its name. Wait until the pod
+is running; this can take up to a few minutes to retrieve the deployer
+image from its registry. You can watch it with:
 +
-====
-----
-$ oc edit scc/privileged
-----
-====
+    $ oc get pod/<pod-name> -w
 +
-And add the following user to the file. The following example uses the `logging`
-project. Be sure to use the project created above in the following example:
+If it seems to be taking too long, you can retrieve more details about the pod and
+any associated events with:
 +
-====
-----
-- system:serviceaccount:logging:aggregated-logging-fluentd
-----
-====
+    $ oc describe pod/<pod-name>
++
+When it runs, check the logs of the resulting pod (`oc logs -f <pod name>`)
```

Comment:
For the 3 instances of `<pod-name>` or `<pod name>` above, use `<pod_name>` (just for style consistency w/ the rest of the docs).